managers/tests/test_unit_managers_apiview.py

```diff
@@ -0,0 +1,125 @@
+import datetime
+
+from rest_framework.reverse import reverse
+from rest_framework.test import APIRequestFactory
+
+from django.test import TestCase
+
+from users.models import CustomUser
+from managers import views
```

Comment:
Alpha ;)

Reply:
@rwrzesien You mean alphabetical order? Without grouping them for exterior ones, django and from each app?

Reply:
Done